zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/schema/ElasticsearchEngineClientIT.java

```diff
@@ -101,6 +112,36 @@ void shouldCreateIndexTemplateCorrectly() throws IOException {
         indexTemplates.getFirst().indexTemplate().template().mappings(), "/mappings.json");
   }
 
+  @Test
+  void shouldWarnIfTryingToCreateExistingTemplate() {
+    // given
+    final var indexTemplate =
+        SchemaTestUtil.mockIndexTemplate(
+            "index_name",
+            "test*",
+            "alias",
+            Collections.emptyList(),
+            "template_name",
+            "/mappings.json");
+
+    final var settings = new IndexSettings();
+    elsEngineClient.createIndexTemplate(indexTemplate, settings, true);
+
+    // when
+    elsEngineClient.createIndexTemplate(indexTemplate, settings, true);
+
+    // then
+    assertThat(logWatcher.list.stream())
+        .anyMatch(
+            log ->
+                log.getLevel().equals(Level.DEBUG)
+                    && log.getFormattedMessage()
+                        .contains(
+                            String.format(
+                                "Did not create index template [%s] as it already exists",
+                                indexTemplate.getTemplateName())));
```

Comment:
💭 I wouldn't assert a debug or a warning log to be honest. That should not be an expected behavior from an API that `it logs`. 
I'd rather assert here that it does not fail. And then in unit tests, and verify it doesn't call `putIndexTemplate` if `exists` returned true. (if we decided to implement it this way).

Logging is/should be hidden implementation details.

But I'd leave that up to you to decide